src/main/scala/uk/gov/hmrc/initrepository/Main.scala

```diff
@@ -95,18 +96,21 @@ object Main {
         } else {
           root.setLevel(Level.INFO)
         }
-        start(config.repoName, config.teamName, config.repoType)
+
+        val webhookUrl = Properties.envOrNone("webhookUrl")
```

Comment:
@andrewgee Could webhook URL be a command line parameter like the other config options please? Thanks

@charleskubicek